app/webpacker/css/admin/components/messages.scss

```diff
@@ -34,19 +34,29 @@
 
   .flash {
     padding: 18px;
-    text-align: center;  
+    text-align: center;
     font-size: 120%;
     color: $color-1;
     font-weight: 600;
     margin-top: 0;
-  
-    &.notice  { background-color: rgba($color-notice,  0.8) }
-    &.success { background-color: rgba($color-success, 0.8) }
-    &.error   { background-color: rgba($color-error,   0.8) }
+
+    &.notice {
+      background-color: rgba($color-notice, 0.8);
+    }
+    &.success {
+      background-color: rgba($color-success, 0.8);
+    }
+    &.error {
+      background-color: rgba($color-error, 0.8);
+    }
```

Comment:
Arguably these lines are now less readable, but Prettier doesn't provide any options to choose a different format.

Reply:
> but Prettier doesn't provide any options to choose a different format.

For me, this is the force of Prettier: to be (almost) full opinionated.